pkg/faketikv/task.go

```diff
@@ -179,6 +180,16 @@ func (t *transferLeader) Step(r *RaftEngine) {
 	t.finished = true
 	r.SetRegion(region)
 	r.recordRegionChange(region)
+	fromPeerID := t.fromPeer.GetId()
+	toPeerID := t.peer.GetId()
+	_, ok := r.schedulerStats.taskStats.transferLeader[fromPeerID]
```

Comment:
Can we let the Stats just `Observe` the task and the statistical behavior is handled in `observe`？